lte/gateway/python/magma/pipelined/app/ipv6_solicitation.py

```diff
@@ -90,11 +91,15 @@ def _get_config(self, config_dict):
             ipv6_src = get_link_local_ipv6_from_if(s1_interface)
             if ipv6_src is None:
                 ipv6_src = self.DEFAULT_LINK_LOCAL_ADDR
-
+        ng_service = config_dict.get('enable5g_features', None)
+        logging.info("enable5g")
+        logging.info(ng_service)
         return self.IPv6RouterConfig(
             ipv6_src=ipv6_src,
             ll_addr=config_dict['virtual_mac'],
             prefix_len=self.DEFAULT_PREFIX_LEN,
+            ng_service_enabled=ng_service,
```

Comment:
`classifier_controller_id` is only used if `ng_service_enabled` is true. Should these configs be merged?

Reply:
Fixed it.